driverstation.py

```diff
@@ -34,6 +34,8 @@ def connectionListener(connected, info):
     """
     #print(info, "; Connected=%s" % connected)
     logging.info("%s; Connected=%s", info, connected)
+    global hasCommunication
```

Comment:
Can't we use the state of the networktables connection (the server is on the robot) to determine if we have communication?        Perhaps it is a bit of work.    Maybe a good issue for someone to work on.